proposals/0216-widget-support.md

```diff
@@ -323,8 +535,47 @@ After investigating impact to SDL Core, the impact is expected to be minor. Glob
 2. Changes to State Controller to allow HMI level transitions for widgets and screens
 3. Changes to Request Controller as RPCs from one app can be addressed to different screens
 
+T screen managers should be refactored to read screen capabilities notifications as well as the deprecated parameters.
+
 ## Alternatives considered
 
-To reduce complexity on the head unit, screen duplication can be supported by the SDL libraries. This could allow more flexibility to  display duplication for existing screens. However this increases the number of RPCs to be sent by the app and the user might see a delay in screen updates (duplication not synchronized).
+To reduce complexity on the head unit, screen duplication can be supported by the SDL libraries. This could allow more flexibility to display duplication for existing screens. However this increases the number of RPCs to be sent by the app and the user might see a delay in screen updates (duplication not synchronized).
 
 In order to reduce state machine complexity for widget HMI levels on SDL core side, the HMI can take control of HMI level transitions for widgets. However this might cause different behavior per HMI implementation which could be confusing to developers as the behavior is not consistent.
+
+Instead of only having display capability type it is possible to separate display capability and screen capability in `SystemCapability`. This may allow some benefits avoiding repeating display data when screens have changed. Also it may allow to notify changes only to a specific screen, instead of repeating all screens data.
```

Comment:
Update to:
Instead of only having display capability **types** it is possible to separate display capability and screen capability in `SystemCapability`. This may allow some benefits avoiding repeating display data when screens have changed. **Also,** it may allow **notification** **changes only specific **to** screens**, instead of repeating all screens data.